mex/editor/edit/main.py

```diff
@@ -1,12 +1,64 @@
 import reflex as rx
 
-from mex.editor.edit.state import EditState
+from mex.editor.edit.state import EditableField, EditablePrimarySource, EditState
 from mex.editor.layout import page
 
 
+def editable_value(value: str) -> rx.Component:
+    """Return a single card for editing one value."""
+    return rx.card(
+        value,
+        style={"width": "30vw"},
+    )
+
+
+def editable_primary_source(model: EditablePrimarySource) -> rx.Component:
+    """Return a horizontal grid of cards for editing one primary source."""
+    return rx.hstack(
+        rx.card(
+            model.name,
+            style={"width": "15vw"},
+        ),
+        rx.vstack(
+            rx.foreach(
+                model.values,
+                editable_value,
+            )
+        ),
+    )
+
+
+def editable_field(model: EditableField) -> rx.Component:
+    """Return a horizontal grid of cards for editing one field."""
+    return rx.hstack(
+        rx.card(
+            model.name,
+            style={"width": "15vw"},
+        ),
+        rx.foreach(
+            model.values,
+            editable_primary_source,
```

Comment:
it's because of how the edit UI is structured:
- each edit page has a list of field sections (depending on which entity type you are editing)
  - each field section has a list of primary-sources (depending on how many extracted items will be merged / have the same stable target id)
    - each primary-source section has a list of concrete values (because even from one primary source, you might extract multiple values)

i changed the name of the "values" attribute here ffba2e5. hopefully that's a bit clearer